composer.json

```diff
@@ -11,12 +11,19 @@
             "email": "stof@notk.org"
         }
     ],
-    "require": {
-        "php": ">=5.3.3",
-        "phpspec/prophecy": "~1.3"
+    "config": {
+        "platform": {
+            "php": "7.3.0"
```

Comment:
Should be removed

Reply:
May I ask why?

Reply:
when the CI runs on PHP 7.4, I want composer to resolve dependencies based on PHP 7.4 to match what would happen for our PHP 7.4 users.